### ISSUE TITLE
can i use static library and use GPU to calculate vector？

### ISSUE THREAD

**Issue by deadspacewii** [DataIntelligenceCrew/go-faiss#21]

**Comment by p-ouellette** [comment]
Yes, you should be able to use static linking. You can pass the libraries to the linker with the `CGO_LDFLAGS` environment variable. I don't have detailed instructions as I haven't tried this myself.

You may want to check out this fork: https://github.com/AnyVisionltd/go-faiss. It looks like they added GPU support.

**Comment by deadspacewii** [comment]
i compile static link and occur this error ，how can i deal with it
![image](https://user-images.githubusercontent.com/17780173/204206792-d4d340ae-dfbb-4fcb-9989-b415ba668d2d.png)
